Disable threaded eval function on Cygwin since it's currently
buggy, and not strictly needed in the first place since we don't
build giac with FLTK support; see https://trac.sagemath.org/ticket/27385
--- a/src/icas.cc	2018-12-22 17:08:24.000000000 +0100
+++ b/src/icas.cc	2019-03-06 14:38:19.814030200 +0100
@@ -160,7 +160,7 @@
 using namespace giac;
 #define STDIN_FILENO 0
 namespace xcas {
-  /*
+#ifdef __CYGWIN__
   void icas_eval(giac::gen & g,giac::gen & gg,int & reading_file,std::string &filename,giac::context * contextptr){
     gg=protecteval(g,10,contextptr);
   }
@@ -168,7 +168,7 @@
   bool fltk_view(const giac::gen & g,giac::gen & ge,const std::string & filename,std::string & figure_filename,int file_type,const giac::context *contextptr){
     return false;
   }
-  */
+#endif
 }
 
 #else
